src/App.jsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] as the previous CR already mentioned "As required by Microverse, you are not meant to use any linters disable rule on your code base unless otherwise stated. The linters are there to make sure your code follows the best practices, I'll suggest you remove these disable rules and fix the issues."